tests/integration/suite/actors/reminders/scheduler/crud.go

```diff
@@ -0,0 +1,168 @@
+/*
+Copyright 2024 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or impliei.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package scheduler
+
+import (
+	"context"
+	"net/http"
+	"strconv"
+	"sync/atomic"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	rtv1 "github.com/dapr/dapr/pkg/proto/runtime/v1"
+	"github.com/dapr/dapr/tests/integration/framework"
+	"github.com/dapr/dapr/tests/integration/framework/process/daprd"
+	prochttp "github.com/dapr/dapr/tests/integration/framework/process/http"
+	"github.com/dapr/dapr/tests/integration/framework/process/placement"
+	"github.com/dapr/dapr/tests/integration/framework/process/scheduler"
+	"github.com/dapr/dapr/tests/integration/framework/util"
+	"github.com/dapr/dapr/tests/integration/suite"
+)
+
+func init() {
+	suite.Register(new(crud))
+}
+
+type crud struct {
+	place     *placement.Placement
+	scheduler *scheduler.Scheduler
+
+	daprd        *daprd.Daprd
+	methodcalled atomic.Uint32
+}
+
+func (c *crud) Setup(t *testing.T) []framework.Option {
+	handler := http.NewServeMux()
+	handler.HandleFunc("/dapr/config", func(w http.ResponseWriter, r *http.Request) {
+		w.Write([]byte(`{"entities": ["myactortype"]}`))
+	})
+	handler.HandleFunc("/healthz", func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(http.StatusOK)
+	})
+	handler.HandleFunc("/actors/myactortype/myactorid/method/remind/xyz", func(http.ResponseWriter, *http.Request) {
+		c.methodcalled.Add(1)
+	})
+	handler.HandleFunc("/actors/myactortype/myactorid/method/foo", func(http.ResponseWriter, *http.Request) {})
+
+	c.scheduler = scheduler.New(t)
+	srv := prochttp.New(t, prochttp.WithHandler(handler))
+	c.place = placement.New(t)
+	c.daprd = daprd.New(t,
+		daprd.WithInMemoryActorStateStore("mystore"),
+		daprd.WithPlacementAddresses(c.place.Address()),
+		daprd.WithSchedulerAddresses(c.scheduler.Address()),
+		daprd.WithAppPort(srv.Port()),
+	)
+
+	return []framework.Option{
+		framework.WithProcesses(c.scheduler, c.place, srv, c.daprd),
+	}
+}
+
+func (c *crud) Run(t *testing.T, ctx context.Context) {
+	c.scheduler.WaitUntilRunning(t, ctx)
+	c.place.WaitUntilRunning(t, ctx)
+	c.daprd.WaitUntilRunning(t, ctx)
+
+	client := util.HTTPClient(t)
+
+	daprdURL := "http://localhost:" + strconv.Itoa(c.daprd.HTTPPort()) + "/v1.0/actors/myactortype/myactorid"
```

Comment:
I think that's a valid/cleaner change unless my syntax is wrong
```suggestion
	daprdURL := "http://" + c.daprd.HTTPAddress()  + "/v1.0/actors/myactortype/myactorid"
```